mono/metadata/object-internals.h

```diff
@@ -388,7 +388,7 @@ struct _MonoInternalThread {
 	MonoThreadHandle *handle;
 	gpointer native_handle;
 	gpointer unused3;
-	gunichar2  *name;
+	char       *name_utf8;
 	guint32	    name_len;
```

Comment:
You could remove `name_len` too with that change.

Reply:
Its meaning is less changed but ok.

Reply:
Oh, remove? Would prefer not. Rename, ok. Nul terminated strings are a bad data structure. Strlen is slow.